packages/vitepress-plugin-git-changelog/src/client/components/Contributors.vue

```diff
@@ -83,7 +83,7 @@ watch(commits, async (newCommits) => {
 </script>
 
 <template>
-  <h2 :id="t('contributors.title')">
+  <h2 :id="t('contributors.titleId')">
```

Comment:
Nice catch.

Reply:
As a convention, heading id is kebab-case naming and all characters are lower case.

And user can customize it so we use another key called `titleId`.